hera_mc/cm_active.py

```diff
@@ -205,23 +205,42 @@ def load_geo(self, at_date=None):
             key = cm_utils.make_part_key(ageo.station_name, None)
             self.geo[key] = ageo
 
-    def revs(self, hpn):
+    def revs(self, hpn, exact_match=False):
         """
-        Returns a list of active revisions for a given hpn.
+        Returns a list of active revisions for given hpn.
 
         Parameters
         ----------
-        hpn : str
-            A HERA part number (not a list)
+        hpn : str or list
+            A HERA part number, or list of part numbers or fractions thereof.
+        exact_match : bool
+            Flag to look for exact matches to part numbers or not.
 
         Returns
         -------
         list
-            List of revisions for supplied hpn.  Typically one element.
+            List of revision Namespaces for supplied hpn.  Can show with cm_revisions.show_revisions
         """
-        hpn_revs = []
-        hpn = hpn.upper()
-        for part in self.parts.values():
-            if hpn == part.hpn.upper():
-                hpn_revs.append(part.hpn_rev)
-        return hpn_revs
+        from argparse import Namespace
+        hpn = [x.upper() for x in cm_utils.listify(hpn)]
+        rev_dict = {}
+        for hloop in hpn:
+            rev_dict[hloop] = {}
+            for part in self.parts.values():
+                phup = part.hpn.upper()
+                use_this_one = (phup == hloop) if exact_match else phup.startswith(hloop)
```

Comment:
what if multiple parts start with the specified string?

Reply:
If there are multiple parts, it lists them all

Reply:
ok. maybe clarify that in the docstring?

Reply:
Done